src/libraries/System.Diagnostics.StackTrace/tests/StackFrameTests.cs

```diff
@@ -123,6 +123,7 @@ public static IEnumerable<object[]> ToString_TestData()
 
         [Theory]
         [ActiveIssue("https://github.com/mono/mono/issues/15186", TestRuntimes.Mono)]
+        [ActiveIssue("TODOTODOTODO", typeof(PlatformDetection), nameof(PlatformDetection.IsNativeAot))]
```

Comment:
Now I remember why I didn't commit this last night. No merge before this is replaced.